clone_repos.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+MIRROR="https://github.com"
+
+function install_rpm () {
+    yum install $1
+}
+
+function install_git () {
+    git clone $MIRROR/$2 -b $3 $1
+    cd $1
+    python setup.py install
+    cd ..
+}
+
+
+# Note: versions are train.  We should try stein...
+
+install_rpm python36-pytz
+
+# epel? :(
+install_rpm python36-pbr
+# pbr!=2.1.0,>=2.0.0 # Apache-2.0
+
+# epel? :(
+install_rpm python36-sqlalchemy
+# SQLAlchemy!=1.1.5,!=1.1.6,!=1.1.7,!=1.1.8,>=1.0.10 # MIT
+
+# epel? :(
+install_rpm alembic
```

Comment:
Let's not fetch any python dependencies from RPM, it can create a mess. pbr may be fine, but anything we explicitly import is not.